src/searcher.cpp

```diff
@@ -2758,6 +2805,7 @@ size_t Searcher::mem_used() const
     size_t mem = HyperEngine::mem_used();
     mem += var_act_vsids.capacity()*sizeof(uint32_t);
     mem += var_act_maple.capacity()*sizeof(uint32_t);
+    mem += lit_act_lsids.capacity()*sizeof(uint32_t);
```

Comment:
It's sizeof double actually... and all the rest is wrong as well, what I wrote :D They are double too! Damn, that's a bug! Please fix all :)